### PR TITLE
[Merged by Bors] - chore: remove superfluous `intro`s

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Preadditive/Biproducts.lean [hunk 1]
```diff
@@ -151,7 +151,6 @@ def biconeIsBilimitOfLimitConeOfIsLimit {f : J → C} {t : Cone (Discrete.functo
     IsLimit.ofIsoLimit ht <|
       Cones.ext (Iso.refl _)
         (by
-          rintro ⟨j⟩
           aesop_cat)
```
Comment (grunweg) [inline Mathlib/CategoryTheory/Preadditive/Biproducts.lean:154]: Can you write this on one line now, as `(by aesop_cat)`?

**Comment by grunweg** [comment]
!bench

**Comment by grunweg** [comment]
FYI, the `awaiting-review` label was recently removed: you can simply omit it now. ([See zulip](https://leanprover.zulipchat.com/#narrow/stream/113486-announce/topic/Changes.20to.20the.20.23queue) for details.)

**Review by grunweg — COMMENTED** [review]
Thanks for your PR! These changes look good; I have one follow-up request.
Out of paranoia, I'd like to see if this makes any difference performance-wise; otherwise, this is good to go.

**Comment by grunweg** [comment]
No performance effect - if you fix the nit, this is ready to be merged.

File: Mathlib/CategoryTheory/Preadditive/Biproducts.lean (hunk 1 above)
Comment (mo271) [reply Mathlib/CategoryTheory/Preadditive/Biproducts.lean]: done, I formatted it to fit everything on one line

**Comment by grunweg** [comment]
Thanks again for your contribution!
maintainer merge

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**